### DISCUSSION TITLE
Umi Max 的命令行 max 很慢，体验较差，可否优化？

### DISCUSSION THREAD

**Discussion by doornot** [umijs/umi#10642] [Q&A]
例如 用 `pnpm max g` 生成模版文件 或者 `pnpm max -h` 查看帮助信息，大概都要等要7秒左右命令行才会有反馈。

有尝试过删除 umi config，会稍快2-3秒。

**Comment by fz6m** [comment]
`max g` 已经跳过了 config 读取最耗时的校验阶段，考虑下是不是插件太多了导致的，提供一个最小复现帮助排查问题。

**Reply by doornot** [reply]
最小复现demo: https://github.com/doornot/umi-max-demo @fz6m

**Reply by fz6m** [reply]
在 #10645 跟进。

同时 `utils/theme.ts` 这么写可以优化一点性能：

```ts
import theme from 'antd/lib/theme';

const { defaultAlgorithm, defaultSeed } = theme

export const mapToken = defaultAlgorithm(defaultSeed);
```

**Reply by doornot** [reply]
好的 感谢 @fz6m
